cpp/cmake/thirdparty/get_raft.cmake

```
@@ -14,24 +14,35 @@
 # limitations under the License.
 #=============================================================================
 
-function(find_and_configure_raft VERSION)
+function(find_and_configure_raft)
 
-    rapids_cpm_find(raft ${VERSION}
-      GLOBAL_TARGETS raft::raft
-      BUILD_EXPORT_SET    cugraph-exports
-      INSTALL_EXPORT_SET  cugraph-exports
+    set(oneValueArgs VERSION FORK PINNED_TAG)
+    cmake_parse_arguments(PKG "${options}" "${oneValueArgs}"
```

Can you update to the following:
```suggestion
    cmake_parse_arguments(PKG "" "${oneValueArgs}" "" ${ARGN})
```
The current approach would use cause unexpected parsing to occur if global variables `options` or `multivalueArgs` existed. ( Or you can set those variables to empty strings  )